unified_range/utils.py

```diff
@@ -87,9 +91,11 @@ def transform_to_semver(unified_spec: str) -> str:
                 lt_lte = operators["lt"]
                 if restriction.has_inclusive_upper:
                     lt_lte = operators["lte"]
+                maybe_comma = ',' if comma_separated else ''
```

Comment:
can we find a better name for that? maybe we can change the logic to be with regular if else?